app/views/users/show.html.erb

```diff
@@ -15,20 +15,29 @@
 
   <div class="card-user">
     <h1>My dogs</h1>
-    <% @dogs.each do |dog| %>
-      <%= link_to dog[:name], dog_path(dog), class: "links-sessions" %>
-      <%= cl_image_tag dog.photo, width: 200, height: 150, crop: :fill %>
-    <% end %>
+    <div class="cards">
+      <% @dogs.each do |dog| %>
+        <div class="card-dog">
+          <h3><%= link_to dog[:name].upcase, dog_path(dog), class: "links-sessions" %></h3>
+          <%= cl_image_tag dog.photo, width: 200, height: 150, crop: :fill %>
+        </div>
+      <% end %>
+    </div>
   </div>
 
 
   <div class="card-user">
     <h1>My bookings</h1>
-    <% @bookings.each do |booking| %>
-      <p>Date: <%= booking[:date] %></p>
-      <p>Description: <%= booking[:description] %></p>
-      <%= link_to 'More details', booking_path(booking), class: "links-sessions" %>
-    <% end %>
+    <div class="cards">
+      <% @bookings.each do |booking| %>
+      <div class="card-dog">
+        <h3><%= booking.dog.name.upcase %></h3>
```

Comment:
You are making a DB request in the view.
You should be passing an @instance variable from the controller